database/cockroachdb/TUTORIAL.md

```diff
@@ -26,7 +26,7 @@ export COCKROACHDB_URL=cockroachdb://cockroach:@localhost:26257/example?sslmode=
 You can find further description of database URLs [here](README.md#database-urls).
 
 ## Create migrations
-Let's create table called `users`:
+Let's create a table called `users`:
```

Comment:
should fix this typo in the commit where it was introduced using git rebase

Reply:
Ok